server/routes/board.js

```diff
@@ -86,8 +86,17 @@ router.put('/', (req, res) => {
   // update board item
 });
 
-router.delete('/', (req, res) => {
-  // delete board item
+router.delete('/cards', (req, res) => {
```

Comment:
So put and delete should take in a path parameter

`/cards/:id`

And this is where you get the id of the card from using `req.params.id`

Reply:
You don't need to send a body for delete.